scripts/keystone_data.sh

```diff
@@ -194,9 +194,9 @@ if [[ "$ENABLED_SERVICES" =~ "n-cpu" ]]; then
     get_or_create_endpoint \
         "placement" \
         "RegionOne" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/"
+        "http://$SERVICE_HOST:8778/" \
```

Comment:
I'm not sure this part of the change is correct, but I'm a bit confused about what upstream wanted to do. there is apparently a mismatch what is documented in the install guide and what the rest of openstack does. 

devstack seems to deploy this on the standard port (80 or 443) under the /placement prefix.  various other things are defaulting to the port 8780, but there is documentation in the admin guide saying that 8778 is taken by senlin. I would say this is inconsistent at best, and the install guide is actually wrong on that.